resources/user.py

```diff
@@ -155,7 +153,7 @@ def post(self):
         if user and user.archived:
             return {"message": "Invalid user"}, 403
 
-        if user and check_pw(data['password'], user.password):
+        if user.check_pw(data['password']):
```

Comment:
It seems like this should be `if user and user.check_pw(...)` - Created ticket codeforpdx/dwellingly-app/issues/427 to deal with this.